IDE/Renesas/e2studio/RA6M3G/include.am

```diff
@@ -0,0 +1,42 @@
+# vim:ft=automake
+# included from Top Level Makefile.am
+# All paths should be given relative to the root
+
+EXTRA_DIST+= IDE/Renesas/e2studio/RA6M3G/benchmark-wolfcrypt/common/.gitkeep
```

Comment:
We probably don't need to include .gitkeep in our release packages, right?  See if output looks as expected when running "make dist" without including .gitkeep files here.  Applies to all .gitkeep files in this file.

Reply:
https://github.com/wolfSSL/wolfssl/blob/91a9117e1b7e7ca316299cef6dfc2243aa127651/IDE/Renesas/e2studio/RA6M3G/include.am#L2-L5

Thanks, I removed all '.gitkeep' files from the include.am